### ISSUE TITLE
[question] federation of longer posts as Article instead of Note?

### ISSUE THREAD

**Issue by carbontwelve** [superseriousbusiness/gotosocial#604]
Prompted by the lack of an opinionated authoring frontend I have been thinking of alternative _future_ uses of GoToSocial. For example a blogging front end where posts are in markdown and the larger ones federated as Article instead of Note.

**Comment by tsmethurst** [comment]
Yes! This is similar to an idea I had (but not sure if I wrote it down anywhere) to provide different templates for different post types, and let the user select from them when posting :)

**Comment by skedastically** [comment]
Coming from [GoBlog](https://goblog.app), I support this too. Other relevant info that points to this:

- `Articles` are generally meant for syndication back to the homepage (from this [discussion](https://socialhub.activitypub.rocks/t/resolving-the-note-vs-article-distinction/258/8)), and
- Mastodon displays non-`Notes` typed posts as a URL with a header (from this [issue](https://github.com/mastodon/mastodon/issues/8892)).

I would like to federate my blogposts as links back to my original GTS instance, and let my short Notes fill the feed. If this is implemented, I would tolerate giving up [table-formatting](https://github.com/superseriousbusiness/gotosocial/pull/116) as well!

**Comment by daenney** [comment]
We've migrated to Codeberg. Your issue has been imported as https://codeberg.org/superseriousbusiness/gotosocial/issues/604.
